portfolio/src/main/webapp/woof.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta charset="UTF-8">
+	<title>My Portfolio</title>
+	<link rel="stylesheet" href="style.css">
+	<script src="script.js"></script>
+</head>
+
+<body>
+    <h1> Woof. </h1>
+	<img src="/images/pet/IMG-0057.JPG" alt="Beef jerky loving dog" height="500">
```

Comment:
I noticed that you only used IMG-0057, but you've uploaded several other images as well. It's best to keep pull requests small, so let's only upload the photos that you need in this PR.